sample.env

```diff
@@ -2,3 +2,5 @@ ESLINT_NO_DEV_ERRORS=true
 TSC_COMPILE_ON_ERROR=true
 REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=http://localhost:31300
 REACT_APP_UNCHAINED_ETHEREUM_WS_URL=ws://localhost:31300
+REACT_APP_UNCHAINED_BITCOIN_HTTP_URL=tbd
```

Comment:
These are set to tbd because we dont yet know what they will point to locally when running eth and btc at the same time

Reply:
i assume so since kevin recently set up that ethereum ws url above